src/frontend/Shared/components/Sidebar.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { connect} from 'react-redux';
+import { increment } from '../actions';
+import {GlobalState} from "../reducers";
+import './style.css'; 
+
+class Sidebar extends React.Component {
+
+    buttonCollapse = () => {
+        let element = document.getElementById("mySidebar");
+        if (element) {
+            element.style.display = "None";
+        }
+    }
+
+    buttonOpen = () => {
+        let element = document.getElementById("mySidebar");
+        if (element) {
+            element.style.display = "Block";
+        }
+    }
+
+    render() {
+        return (
+            <div>
+                <div id="mySidebar" className="sidebar">
+                <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet"/>
+                <button className="openbtn" onClick={() => this.buttonCollapse()}>&#9776;</button> 
+                <a><img src={require("./test.png")} width="100"height="100"/>
+                <p>Friendsheep</p></a>
+                <a href="#" className="glyphicon glyphicon-home">Home</a>
+                <a href="#" className="glyphicon glyphicon-user">Chats</a>
+                <a href="#" className="glyphicon glyphicon-cog">Setting</a> 
```

Comment:
We can also extract this as a separate component so we will have a better  structure.

Reply:
OK, I extract a new component called MenuItem